database/insertData/03_template_C_userRegistration.js

```diff
@@ -181,6 +181,14 @@ exports.queries = [
                     operator: "objectProperties"
                     children: ["applicationData.responses.Q5.hash"]
                   }
+                  username_value: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.Q3"]
```

Comment:
Oh I see, this is so it can display the SummaryView on the table.

I wonder if this is not a good case for a jsonb field to collect all of these together. Seems like we're going to have a lot of extraneous columns just to hold response data, so perhaps they should all go into one column called "application_responses", and have a field for each response you want to keep.

Reply:
This one in particular was an artefact of testing.

> I wonder if this is not a good case for a jsonb field to collect all of these together. Seems like we're going to have a lot of extraneous columns just to hold response data, so perhaps they should all go into one column called "application_responses", and have a field for each response you want to keep.

I am not sure if column quantity is an issue ? (still same amount of data right ? it's just hidden in a jsonb field)

I think it's much easier to see what 'things' are in the outcome by looking at the table vs looking at possible values inside jsonb field of one of the records.

Anyways I've removes saving username_value:

d2de7a94364f1d5b9509c9c2437320bb337d8e7d